### DISCUSSION TITLE
How to free-free Modal Analysis in ROSS

### DISCUSSION THREAD

**Discussion by Adonay4** [petrobras/ross#1082] [Q&A]
Hello, I was wondering how to run a free-free modal analysis in ROSS. I am new to ROSS and rotor dynamics, I checked the documentation online but could not find a way to do it. I will be grateful if anybody has an answer or leads.
Thanks

**Comment by jguarato** [comment]
Hi @Adonay4 !

You can assemble the rotor without bearing elements and run the modal analysis for it. However, we have noticed that there are some issues associated with this type of analysis. We are investigating and hope to solve this soon.

**Reply by Adonay4** [reply]
Thank you @jguarato
